src/audio/google/google_rtc_audio_processing.c

```diff
@@ -42,6 +42,11 @@
 #define GOOGLE_RTC_AUDIO_PROCESSING_FREQENCY_TO_PERIOD_FRAMES 100
 #define GOOGLE_RTC_NUM_INPUT_PINS 2
 #define GOOGLE_RTC_NUM_OUTPUT_PINS 1
+#define ERR_INVALID_REF -1
+#define ERR_MEMCPY_FAIL -2
+#define ERR_INVALID_SRC -3
+#define ERR_INVALID_DST -4
```

Comment:
We use typically -EINVAL or -ENOMEM depending on the case and text description of issue, so not sure if this detail level is needed. They might also cause confusion with errno.h values 1- 4 when reported back.

Reply:
@ShriramShastry NAK, this file as well as most of the SOF use POSIX error codes, please don't invent any new ones and mix them with existing ones.